### DISCUSSION TITLE
List of available properties (proptery_id, values_ids...)

### DISCUSSION THREAD

**Discussion by kapik** [etsy/open-api#776] [Q&A] [answered]
Hey 👋

I  want to add properties to my draft.
I want to add the following properties for example: Art subject, Holiday, Framing, Orientation...

I can see that I need to use `updateListingProperty` and that I need a `property_id`. 

Are the `property_id` in common for everyone?
Where can we find a property listing (enum) to find the `id`?
Same questions for `value_ids`.

Today, to find this info I create a draft on the Etsy interface and I use `getListingProperties` to see what it corresponds to...

Thx for your help 🙂

**Comment by baahar** [comment, marked as answer]
Hi,

No, properties are not universal. They depend on the category of a listing. So for any given category in the seller taxonomy there are associated properties and possible values for those properties.

You can get them via the getPropertiesByTaxonomyId endpoint: https://developers.etsy.com/documentation/reference/#operation/getPropertiesByTaxonomyId

**Reply by kapik** [reply]
ha yes ok I didn't understand it like that. 
Thanks
